src/profile-logic/interval-marker-styles.js

```diff
@@ -8,7 +8,7 @@ export const styles = {
   default: {
     top: 0,
     height: 6,
-    background: 'black',
+    background: colors.ORANGE_50,
```

Comment:
Hey @andrapetronela, the goal isn't to change the `default` but the marker named `GCSlice`. Therefore I think you'll want to add a new object at the end of this list :)